carbonmark-api/src/routes/purchases/get.utils.ts

```diff
@@ -0,0 +1,38 @@
+import { utils } from "ethers";
+import { GetPurchaseByIdQuery } from "src/.generated/types/marketplace.types";
+import { Purchase } from "../../models/Purchase.model";
+import { CreditId } from "../../utils/CreditId";
+
+/** Purchase ids are a txn hash */
+export const isValidPurchaseId = (id?: string | null) => {
+  if (!id) return false;
```

Comment:
```suggestion
export const isValidPurchaseId = (id: string) => {
```

It seems like id can't be null at the call site correct?